src/screens/MemberDetail/MemberDetail.tsx

```diff
@@ -18,19 +19,29 @@ import Loader from 'components/Loader/Loader';
 
 type MemberDetailProps = {
   id: string; // This is the userId
+  from?: string;
+};
+type LocationStateProps = {
```

Comment:
You could have used MemberDetailProps here, why create LocationStateProps which have similar keys ?, this is redundant